AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/InitializeMutation.dfy

```diff
@@ -176,13 +176,12 @@ module {:options "/functionSyntax:4" } InternalInitializeMutation {
         internalInput := input,
         commitment := readItems.MutationCommitment.value);
       if (resumeMutation?) {
-        // TODO-SystemKey :: ResumeMutation will validate SystemKey
         output := ResumeMutation(
           commitment := readItems.MutationCommitment.value,
           index := readItems.MutationIndex,
           logicalKeyStoreName := input.logicalKeyStoreName,
           storage := input.storage, // );//,
```

Comment:
why is the syntax of this line like this?

Reply:
Oh... shoot, I meant to fix that.